setup.cfg

```diff
@@ -1,10 +1,22 @@
+# Configure flake8 for testing pep-8 style compliance
 [flake8]
+# TODO: Change this to "black" default of 88
 max-line-length = 120
```

Comment:
Keep `120` (as in `software` repo) & override `black` default of `88` to this? Or keep the default?

@snova-mattw any inputs?